SQL/0000-00-03-ConfigTables.sql

```diff
@@ -209,7 +210,8 @@ INSERT INTO Config (ConfigID, Value) SELECT ID, "/./i" FROM ConfigSettings WHERE
 INSERT INTO Config (ConfigID, Value) SELECT ID, "false" FROM ConfigSettings WHERE Name="showTransferStatus";
 INSERT INTO Config (ConfigID, Value) SELECT cs.ID, GROUP_CONCAT(mst.Scan_Type) FROM ConfigSettings cs JOIN mri_scan_type mst WHERE cs.Name="tblScanTypes" AND mst.ID=44;
 INSERT INTO Config (ConfigID, Value) SELECT cs.ID, GROUP_CONCAT(mst.Scan_Type) FROM ConfigSettings cs JOIN mri_scan_type mst WHERE cs.Name="tblScanTypes" AND mst.ID=45;
-
+INSERT INTO Config (ConfigID, Value) SELECT cs.ID, "mri_parameter_form" FROM ConfigSettings cs WHERE cs.Name="ImagingBrowserLinkedInstruments";
+INSERT INTO Config (ConfigID, Value) SELECT cs.ID, "radiology_review" FROM ConfigSettings cs WHERE cs.Name="ImagingBrowserLinkedInstruments";
```

Comment:
@driusan 
you made a comment in the other pull request about projects that do not have those default tables;
but notice the 2 lines below this comment, they do insert without doing any conditional checks on table existence, so I did the same in my additions.

Reply:
My comment was just in reference to your comment saying that the lines could removed because there's no way to enter the invalid table names if they don't have them setup, I didn't mean you had to change it.

Reply:
yes, I know, your point is a valid one. I think the table additions as default should be conditional; then I just realized the 2 lines below my changes are non-conditional either although they use the same tables... not sure how/where they are used and if they cause any issues for projects that don't have the tables.
I will now make the patch for existing projects with the proper conditions as suggested by @PapillonMcGill, then come back to this and see if that same change should be made here